ui/apps/platform/src/Components/PatternFly/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -23,7 +23,7 @@ type State =
           errorInfo: ErrorInfo;
       };
 
-class ErrorBoundary extends Component<Props, State> {
+class ErrorBoundaryClass extends Component<Props, State> {
```

Comment:
I actually original had the naming you described but switched it because I wasn't sure if we typically name a file differently from the default export. I went back and forth on this though, so I'm glad to get a second opinion. I'll revert it.